File: Desktop/Tubes-OOP-master/src/Product/Product.cpp
#include "Product/Product.h"


Product::Product(){
    price = 0;
}
Product::Product(int _price){
    price = _price;
}
void Product::sell(){
    //nambahin duit player
    //destruk objek
}
int Product::getPrice() const{
    return price;
}
void Product::setPrice(int _price){
    price = _price;
}

File: EngiJava/core/src/com/engicorp/oop/product/Product.java
package com.engicorp.oop.product;

public abstract class Product{
    protected int price;
    protected String name;

    public Product(int _price, String _name){
        price = _price;
        name = _name;
    }
    public int getPrice(){
        return price;
    }
    public void setPrice(int _price){
        price = _price;
    }
    
    public String getName(){
        return name;
    }
    public void setName(String _name){
        name = _name;
    }
    
}